### PR TITLE
UniversalMediaEngine Release 1.1.0.0

### PR DESCRIPTION
**Pull request by bethoma**
Notes:
	- Added PlayStream API to allow playing of IRandomAccessStream
	  objects

	- Workaround for IoTCore MediaEngine incosistencies
	  (SetSourceFromByteStream requires specific URL input)

### CODE REVIEW

File: samples/basicMediaEngineSample/basicMediaEngineSample/MainPage.xaml.cpp [hunk 1]
```diff
@@ -30,18 +30,18 @@ MainPage::MainPage()
...
-	m_mediaEngine.MediaStateChanged += ref new UniversalMediaEngine::MediaStateChangedHandler(this, &basicMediaEngineSample::MainPage::OnMediaStateChanged);
+	m_mediaEngine.MediaStateChanged += ref new MediaStateChangedHandler(this, &basicMediaEngineSample::MainPage::OnMediaStateChanged);
 	m_mediaEngine.InitializeAsync();
 }
 
 
 void basicMediaEngineSample::MainPage::Button_Click(Platform::Object^ sender, Windows::UI::Xaml::RoutedEventArgs^ e)
 {
-	m_mediaEngine.Play(UrlBox->Text);
+	//m_mediaEngine.Play(UrlBox->Text);
```
Comment (ooeygui) [inline samples/basicMediaEngineSample/basicMediaEngineSample/MainPage.xaml.cpp]: Is this right?

File: samples/speechSynthesisSample/speechSynthesisSample/Package.appxmanifest [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="4474c45b-baf1-4ef5-ae2c-962ca5eafb10"
+    Publisher="CN=bethoma"
```
Comment (ooeygui) [inline samples/speechSynthesisSample/speechSynthesisSample/Package.appxmanifest]: Microsoft

File: samples/speechSynthesisSample/speechSynthesisSample/Package.appxmanifest [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="4474c45b-baf1-4ef5-ae2c-962ca5eafb10" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>speechSynthesisSample</DisplayName>
+    <PublisherDisplayName>bethoma</PublisherDisplayName>
```
Comment (ooeygui) [inline samples/speechSynthesisSample/speechSynthesisSample/Package.appxmanifest]: Microsoft

File: samples/speechSynthesisSample/speechSynthesisSample/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("speechSynthesisSample")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (ooeygui) [inline samples/speechSynthesisSample/speechSynthesisSample/Properties/AssemblyInfo.cs]: Microsoft

**Comment by mwmckee** [comment]
please add "speechSynthesisSample/project.lock.json" to the .gitignore

File: samples/basicMediaEngineSample/basicMediaEngineSample/MainPage.xaml.cpp (hunk 1 above)
Comment (bethoma) [inline samples/basicMediaEngineSample/basicMediaEngineSample/MainPage.xaml.cpp]: nope, good catch

**Comment by mwmckee** [comment]
signoff

**Pull request merged**